src/lib/assets/fonts/index.scss

```diff
@@ -0,0 +1,22 @@
+// Use Inter mixin to set font-display: block.
```

Comment:
This is where `sass` is used - to generate only the fonts needed, and with `font-display: block`.